src/components/organisms/MovieCard/index.tsx

```diff
@@ -19,11 +24,13 @@ const MovieCard: FC<IMovieCard> = ({
   release_date,
 }) => {
   const [isFavourite, setIsFavourite] = useState(false);
+  const [addToFavourites] = useAddToFavouritesMutation();
+  const { refetch } = useGetFavouriteMoviesQuery();
 
-  const addToFavourites = () => {
-    setIsFavourite(!isFavourite);
-
-    //here will be the logic of adding to favourites
+  const handleAddToFavourites = async () => {
+    setIsFavourite(true);
+    await addToFavourites(id);
+    refetch();
```

Comment:
the same as comments above 
no Loding 
no Error handling 
No invalidated tags